WORKSPACE

```diff
@@ -18,6 +19,14 @@ http_archive(
     urls = ["https://github.com/google/googletest/archive/release-1.10.0.zip"],
 )
 
+new_git_repository(
```

Comment:
Let's use `http_archive` instead, this is the best practice -- https://docs.bazel.build/versions/master/external.html#repository-rules